platforms/hyperledger-fabric/charts/install_chaincode/templates/install_chaincode.yaml

```diff
@@ -157,7 +157,7 @@ spec:
           mkdir -p $GOPATH/bin && curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
           cd $GOPATH/src/github.com/chaincode && dep ensure
           ## Installing chaincode on a peer
-          peer chaincode install -n ${CHAINCODE_NAME} -v ${CHAINCODE_VERSION} -p github.com/chaincode/${CHAINCODE_NAME}/${CHAINCODE_MAINDIR}
+          peer chaincode install -n ${CHAINCODE_NAME} -v ${CHAINCODE_VERSION} -p $GOPATH/src/github.com/chaincode
```

Comment:
What if there are multiple chaincodes in the $GOPATH/src/github.com/chaincode/ directory? How will the install chaincode work then? And each chaincode can have a different main directory, as I mentioned in the comment.

Reply:
thanks, got your point, am making the changes accordingly and will update it